lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimation.kt

```diff
@@ -82,7 +82,7 @@ fun LottieAnimation(
 ) {
     val drawable = remember {
         LottieDrawable().apply {
-            enableMergePathsForKitKatAndAbove(true)
+            enableMergePathsForKitKatAndAbove(state.enableMergePaths)
```

Comment:
This shouldn't be set here because it will only capture the initial state. I think it should be something like
```
SideEffect(state.enableMergePaths) {
    drawable.enableMergePathsForKitKatAndAbove = state.enableMergePaths
}
````

Reply:
Hah, totally missed that we're remembering this here.. thanks! I ended up pushing the read down to the `draw` block which lets us skip the recomposition.